### PR TITLE
create get all services endpoint and stub out

### PR DESCRIPTION
**Pull request by MHBauer**
Needed this for my understanding so I stubbed it out. Will use soon.

### CODE REVIEW

File: k8s/service_controller/server/nil/storage.go [hunk 1]
```diff
@@ -79,6 +79,10 @@ func (s *NilServiceStorage) GetBrokerByService(id string) (*model.ServiceBroker,
 	return nil, nil
 }
 
+func (s *NilServiceStorage) AllServices() ([]*model.Service, error) {
```
Comment (duglin) [inline k8s/service_controller/server/nil/storage.go]: Why do we have this if we never call it?

Comment (MHBauer) [inline k8s/service_controller/server/nil/storage.go]: at some point in the past I was thinking of using it for testing or as a template for other storage implementations. Might be more trouble than it's worth.

Comment (duglin) [inline k8s/service_controller/server/nil/storage.go]: can we add it later when needed? Just feels odd if its not used.
We still need the rest of the PR though, right?

Comment (MHBauer) [inline k8s/service_controller/server/nil/storage.go]: won't compile without a full implementation.

Comment (duglin) [inline k8s/service_controller/server/nil/storage.go]: that's why I'm a bit confused. It compiles for me w/o this method. I didn't see any ref to AllServices() in any interface.

Comment (MHBauer) [inline k8s/service_controller/server/nil/storage.go]: I see what you mean. I'll take a closer look.

**Comment by MHBauer** [comment]
@duglin fixed

**Comment by duglin** [comment]
LGTM

**Pull request merged**